### PR TITLE
proof of personhood passport staff only

### PR DESCRIPTION
**Pull request by owocki**
##### Description

a WIP implementation of sybilpassport - the ability to take gitcoin's sybil resistence from site to site, on trustbonus tab: [video](https://share.getcloudapp.com/6quQoOnN)

TODOs
- [x] NFT smart contract
- [x] frontend MVP
- [x] frontend well designed
- [x] tx signature works for NFT
- [ ] 3box/cermaic upload (as opposed to NFT) - in a v2
- [ ] spruce id upload option - in a v2

##### Refers/Fixes

https://github.com/gitcoinco/PersonhoodPassport/projects/1

##### Testing

tested locally, works well!

Screen Shot 2021-03-16 at 7 36 50 PM

### CODE REVIEW

File: app/passport/views.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    abi = json.loads(abi)
+    contract = w3.eth.contract(contract_address, abi=abi)
+
+    #_hash = sp.getMessageHash(player, 0, tokenURI, nonce);
+    message = contract.functions.getMessageHash(player, 0, tokenURI, nonce).call()
+    private_key = settings.PASSPORT_PK
+    #signature = await accounts[0].signMessage(ethers.utils.arrayify(hash));
+    signed_message = w3.eth.account.signHash(message.hex(), private_key=private_key).signature.hex()
```
Comment (owocki) [inline app/passport/views.py:25]: for some reason this signed message isnt working in the contract.

Comment (owocki) [reply app/passport/views.py:25]: https://github.com/gitcoinco/gitcoinpassport/blob/master/test/GitcoinPassport.js#L37-L38

the ethers.js signature works though.  i wonder if im using the wrong signing message!

Comment (owocki) [reply app/passport/views.py:25]: here is the line that is failing https://github.com/gitcoinco/gitcoinpassport/blob/master/contracts/GitcoinPassport.sol#L19-L22

**Comment by thelostone-mc** [comment]
Closing this out cause we got this into https://github.com/gitcoinco/web/pull/8569 which is merged

**Pull request closed without merging**